Assets/Scripts/Config.cs

```diff
@@ -45,6 +45,7 @@ public enum XrSdkMode
         Oculus,
         Wave,
         Pico,
+        Zapbox
```

Comment:
please add a trailing comma so that when we add the next one, we won't need to modify two lines

Reply:
Will action!